src/Services/DocumentSign/SignRequest.php

```diff
@@ -0,0 +1,129 @@
+<?php
```

Comment:
missing doc blocks for class methods, return types and method description for whole file